boards/native/native_posix/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ zephyr_library_include_directories(
   )
 
 if(CONFIG_HAS_SDL)
-	add_subdirectory(${ZEPHYR_BASE}/boards/boards_legacy/${ARCH}/common/sdl/ ${CMAKE_CURRENT_BINARY_DIR}/sdl)
+	add_subdirectory(${ZEPHYR_BASE}/boards/native/common/sdl/ ${CMAKE_CURRENT_BINARY_DIR}/sdl)
```

Comment:
why is this using a path from zephyr base, and not relative to current folder ?
Such as: 
```
	 add_subdirectory(${CMAKE_CURRENT_LIST_DIR}/../common/sdl/ ${CMAKE_CURRENT_BINARY_DIR}/sdl)
```
seems a bit safer.
However, current proposal is also ok, as I don't expect we'll be moving boards around so much more.